code/model/DMSDocument.php

```diff
@@ -914,6 +923,13 @@ public function getCMSFields()
         $fields->add(new TextField('Title', 'Title'));
         $fields->add(new TextareaField('Description', 'Description'));
 
+        $downloadBehaviorSource = array(
+            '' => 'Default ('.Config::inst()->get('DMSDocument', 'default_download_behaviour').')',
```

Comment:
The point is that it's not going to be clear to a CMS user than a selection they've made now (to force download) may change if the dev makes a code change.

This is a security feature and having it able to chance unexpectedly is no good.

From my experience of non-technical CMS users they would not:

1. Understand that the default could change
2. Understand that they could request the default be changed and this have a retrospective effect
3. Have taken this into account for security sensitive documents that had previously been marked to force download (because it was default). ie: they won't have gone and picked the second "force download" option if the first was already selected.

The idea of "I want this document to do whatever the system default is" doesn't exist to me when considering security and opening docs in the browser or not.

I think the chances of a client wanting the system default changed is so rare that the action of a dev having to write an SQL query to do it is an acceptable resolution.

If this is currently a requirement of yours, I'd look into applying this case as an extension onto the object.